### PR TITLE
Dev separate js file

### PR DESCRIPTION
**Pull request by DanRathbun**
This PR simply cut out the script embedded in "inputbox.html" and pasted it into it's own file.
I stuck it the same folder for now. (I can't remember how to create a new folder via the GitHub interface.)

### CODE REVIEW

File: src/input_box/html/inputbox.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+    const KEY_RETURN = 13;
```
Comment (thomthom) [inline src/input_box/html/inputbox.js]: Can you undent this file? The "root" indent was because it followed the HTML indentation.

**Comment by thomthom** [comment]
> (I can't remember how to create a new folder via the GitHub interface.)

When you create a file on GitHub you can type `/` to create folders. For instance, if I create a new file in `htmldialog-inputbox/src/input_box` and type `js/` it will look like this:

image

Btw, have you tried the built-in editor they added? It's activated by tapping `.` when viewing a file:

It will open a web version of VSCode:

image

(Hm..  I don't recall if this is a beta feature or not...)

File: src/input_box/html/inputbox.js (hunk 1 above)
Comment (DanRathbun) [reply src/input_box/html/inputbox.js]: Yes can do this. You want no indent at all ?

**Comment by DanRathbun** [comment]
> Btw, have you tried the built-in editor they added? It's activated by tapping `.` when viewing a file:
> 
> It will open a web version of VSCode:

Oh? I did not know about this. I'll see if I can try this to change the indents.

**Comment by DanRathbun** [comment]
I tried to use the Web VSCode editor to make the changes to the guidelines logo URLs, but it loaded but could "not resolve repository folder". I gave up.

**Comment by thomthom** [comment]
Nice change. 👍

**Pull request merged**